### ISSUE TITLE
With firefox - *.mkv files get renamed to *.webm

### ISSUE THREAD

**Issue by nitschis** [filebrowser/filebrowser#1998]
*.mkv files get downloaded as *.webm, only when using firefox, but not with chrome.

The headers are correct in the file:
```
Format                               : Matroska
Format version                       : Version 4
```

**Comment by ramiresviana** [comment]
I'm unable to reproduce this issue, can you provide a sample file? Keep in mind that FileBrowser does not manipulate the file extension, it tells the browser to use the filename as it is when downloading.

**Comment by nitschis** [comment]
Thank you for your reply. Here is an example: shorturl.at/ltH34

**Comment by nitschis** [comment]
My guess is, that it is a Firefox problem. But what triggers it would be interesting and if we can evade it.

**Comment by ramiresviana** [comment]
The provided example is unavailable, files up to 25 MB can be attached to the comment.

**Comment by nitschis** [comment]
Here is the example file, I only had it shared for 2 days.
I can not attach it, as github does not accept mkv files.

This example I will share for 14 days: shorturl.at/ghlGP

**Comment by ramiresviana** [comment]
The provided URL is not working, you can attach zip files on the comment.

**Comment by nitschis** [comment]
Oh, I am sorry, no idea why the link was removed... Here is the file
[tesfile.zip](https://github.com/filebrowser/filebrowser/files/8968541/tesfile.zip)
:

**Comment by ramiresviana** [comment]
I'm unable to reproduce this on the current Firefox version `101.0.1` using the provided sample. Are you able to preview the video on the browser? Do you have any browser extensions installed?

**Comment by nitschis** [comment]
I have encoded the URL in Base64 and have it shared for 14 day, I hope it works now.
aHR0cHM6Ly9zaGFyZS5ldmVybWluZC5hdC9zaGFyZS9saDFHdXdBaA==

**Comment by nitschis** [comment]
>  Are you able to preview the video on the browser? Do you have any browser extensions installed?

No, unfortunately no h265 support. No plugins, I actually got the report from a user and then tried it my self, and he was correct.

**Comment by ramiresviana** [comment]
This might be related to the MIME type detection, check https://github.com/filebrowser/filebrowser/issues/1439#issuecomment-865988937

**Comment by nitschis** [comment]
I use Linux and the user who reported it uses Windows, pretty sure we all use default settings. Does it happen on your end with my provided link? If yes, the question is what is different with your test server, where this does not happen compared to mine.

**Comment by ramiresviana** [comment]
Your server is responding with `video/webm` as the value of `content-type` header, while on my testing instance the value is `video/x-matroska`. This controls the behavior on the client and is independent of the browser and system.

You should check if your server mime information is correct, on Linux the mime information can be found at `/usr/share/mime/` and `/etc/mime.types` but this may  change on different systems.

**Comment by nitschis** [comment]
Thank you very much for leading me to the right path. My docker container did not have any files for mime-types, I tried editing my swag (ngnix proxy) docker, but that changed nothing.

In the end, I fixed it by migrating the container from:
"hurlenko/filebrowser"
to 
"filebrowser/filebrowser"

Which seems to have the correct mime-types. Thank you so much for everything! I hope this will help someone else in the future, so it wasn't a complete waste of your time.
